components/raftstore-v2/src/operation/query/local.rs

```diff
@@ -86,11 +86,7 @@ where
         self.cached_tablet.latest().unwrap()
     }
 
-    fn get_snapshot(
-        &mut self,
-        _: Option<ThreadReadId>,
-        _: &mut Option<raftstore::store::LocalReadContext<'_, E>>,
-    ) -> Arc<E::Snapshot> {
+    fn get_snapshot(&mut self, _: &mut Option<&mut LocalReadContext<'_, E>>) -> Arc<E::Snapshot> {
```

Comment:
Why changed `&mut Option<LocalReadContext` to ` &mut Option<&mut LocalReadContext` ?

Reply:
Good catch. It does not need to be mut now.